satip/app.py

```diff
@@ -61,7 +61,16 @@
     help="Database to save when this has run",
     type=click.STRING,
 )
-def run(api_key, api_secret, save_dir, history, db_url: Optional[str] = None):
+@click.option(
+    "--use-rescaler",
+    default=False,
+    envvar="USE_RESCALER",
```

Comment:
Would you mind dropping a little section in the readme with these env vars, Just so its easy to look up when runing live?

Reply:
Yep!